pkg/docker/config/config_test.go

```diff
@@ -559,6 +559,7 @@ func TestGetAuthFailsOnBadInput(t *testing.T) {
 }
 
 func TestGetAllCredentials(t *testing.T) {
```

Comment:
Please add tests for the other modified functions as well.

Reply:
>Please add tests for the other modified functions as well.

Are those functions properly tested right now anyway? I believe they are not tested with respect to helpers right now.

Reply:
>I believe they are not tested with respect to helpers right now.

For instance the mock helper does noting for the store operation so you can really assert it written anything when calling the `SetCredentials` function.